tests/autoyast/installation.pm

```diff
@@ -284,7 +284,6 @@ sub run {
             send_key 'alt-o';
         }
         elsif (match_has_tag('linuxrc-start-shell-after-installation')) {
-            @needles = grep { $_ ne 'linuxrc-start-shell-after-installation' } @needles;
```

Comment:
is this intended? if we don't remove the needle it will match several times and send exit several times afterwards, editing bootloader with `x`, because of that I added there as in the rest of the code is doing in this module.

Reply:
Strange:) I did not do this change:) For sure, reverting back. I do not need these changes.